frontend/src/static/js/components/webstatus-line-chart-panel.ts

```diff
@@ -217,10 +225,11 @@ export abstract class WebstatusLineChartPanel extends LitElement {
 
   /**
    * Returns the input for generating chart options.
+   * @param {BrowsersParameter[]} browsers The list of browsers to be displayed.
    * @abstract
    * @returns {{seriesColors: Array<string>; vAxisTitle: string;}} Chart options input.
    */
-  abstract getDisplayDataChartOptionsInput(): {
+  abstract getDisplayDataChartOptionsInput(browsers: BrowsersParameter[]): {
```

Comment:
Going with my above suggestion, I would revert this change.
```suggestion
  abstract getDisplayDataChartOptionsInput(): {
```

Reply:
I'm updating this to use the series type 
i.e. `abstract getDisplayDataChartOptionsInput<S>(browsers: <S>[])`

This is still needed due to the tabbed charts needing to obtain multiple different options.